### DISCUSSION TITLE
OpenCRVS v1.0.1 - Installation

### DISCUSSION THREAD

**Discussion by euanmillar** [opencrvs/opencrvs-core#3604] [Technical Q&A]
If you have any questions regarding installation, first read these [docs](https://documentation.opencrvs.org/setup/3.-installation)

If you still need help, ask here.

**Comment by euanmillar** [comment]
I just released this video which I hope helps with setting up a local, demo development environment of OpenCRVS: https://www.loom.com/share/6e649d5d32464523ac40b5d9c4c750d5

**Reply by baldwinox** [reply]
Thanks. Do you have any video or documentation on the 3 node cluster configuration. I am past setting up the cluster infrastructure which went smooth

**Comment by opencrvsug** [comment]
Hello, Thanks for the great work. I cant seem to wrap my mind around the error. In the attachment. I have probably done all and above what is stated in the documentation but no results for me. I know its something small i am missing out but i cant wrap my head around it. Any help would suffice. Thanks 

<img width="1031" alt="Screenshot 2022-08-27 at 11 07 41" src="https://user-images.githubusercontent.com/111888678/187021353-f72018b8-94ca-4dc6-9352-09f24f5254f8.png">

**Reply by rikukissa** [reply]
Hey! I solved this recently by first SSHing into the server I'm deploying to, then running `ssh-keyscan -H my-opencrvs-instance-url.opencrvs.org`. Replace `my-opencrvs-instance-url.opencrvs.org` with the URL to your instance.  Then I took the return value of that command and placed it into the `KNOWN_HOSTS` secret. Hope this helps.

**Reply by opencrvsug** [reply]
Thank you Riku. It worked like a charm!

**Comment by baldwinox** [comment]
Wow Riku! Thanks for the prompt response. I reckon i am close to finishing
the installation. I know i have asked your help already but i ask again and
i will be grateful. From the error i am getting, my deployment is failing
due to permission issues. I am used an account that has root access but is
not root. ( the ssh keys,known hosts etc are derived from this account)

From the error, could it be that the i must use a root user account as it
seems hard coded (*root@***: Permission denied (publickey,password)*

Thanks

**Reply by opencrvsug** [reply]
This is the issue from my post above. I want to know if i can use any other user other than root. 

<img width="1000" alt="Screenshot 2022-08-27 at 16 38 00" src="https://user-images.githubusercontent.com/111888678/187033941-319efb91-233f-4093-a591-e414f714d2b7.png">

**Reply by rikukissa** [reply]
Happy to help :) Have you tried setting `SSH_USER` environment variable to the username you want to deploy with?

**Comment by opencrvsug** [comment]
Hello @rikukissa  and lastly regarding Installation as you can see from the attached screenshot i can access the openhim admin console.. totally awesome! even though i cannot login. the default password in the documentation and shared in the discussion group is not working either. Is there a particular one for 3 server QA setup ?  Docker also gives me this error meaning i must have configured it at one point but i dont recall when since it was done via the git "deploy" ACTION. Is there a way this password can be reset. Its not even giving me the option with the heartbeat link. Thanks 

<img width="1202" alt="Screenshot 2022-09-02 at 12 40 45" src="https://user-images.githubusercontent.com/111888678/188113276-315f8f42-d343-43b2-b7c5-03dee8553cb2.png">

**Reply by opencrvsug** [reply]
Please ignore. I found the password to be "openhim-password" :-)
